README.md

```diff
@@ -1,24 +1,25 @@
 # Wagtail A/B Testing
 
-[![Version](https://img.shields.io/pypi/v/wagtail-ab-testing.svg?style=flat)](https://pypi.python.org/pypi/wagtail-ab-testing/)
-[![License](https://img.shields.io/badge/license-BSD-blue.svg?style=flat)](https://opensource.org/licenses/BSD-3-Clause)
-[![Test & Lint](https://github.com/wagtail-nest/wagtail-ab-testing/actions/workflows/test.yml/badge.svg)](https://github.com/wagtail-nest/wagtail-ab-testing/actions/workflows/test.yml)
+[![License: BSD-3-Clause](https://img.shields.io/badge/license-BSD-blue.svg?style=flat)](https://opensource.org/licenses/BSD-3-Clause)
+[![Build status](https://github.com/wagtail-nest/wagtail-ab-testing/actions/workflows/test.yml/badge.svg)](https://github.com/wagtail-nest/wagtail-ab-testing/actions/workflows/test.yml)
 [![codecov](https://img.shields.io/codecov/c/github/wagtail-nest/wagtail-ab-testing?style=flat)](https://codecov.io/gh/wagtail-nest/wagtail-ab-testing)
+[![Version](https://img.shields.io/pypi/v/wagtail-ab-testing.svg?style=flat)](https://pypi.python.org/pypi/wagtail-ab-testing/)
+[![Monthly downloads](https://img.shields.io/pypi/dm/wagtail-ab-testing.svg?logo=Downloads)](https://pypi.python.org/pypi/wagtail-ab-testing/)
 
 Wagtail A/B Testing is an A/B testing package for Wagtail that allows users to create and manage A/B tests on pages through the Wagtail admin.
 
 Key features:
 
- - Create an A/B test on any page from within Wagtail
- - Tests using page revisions (no need to create separate pages for the variants)
- - It prevents users from editing the page while a test is in progress
- - Calculates confidence using a Pearson's chi-squared test
+-   Create an A/B test on any page from within Wagtail
+-   Tests using page revisions (no need to create separate pages for the variants)
+-   It prevents users from editing the page while a test is in progress
+-   Calculates confidence using a Pearson's chi-squared test
 
 [Changelog](https://github.com/torchbox/wagtail-ab-testing/blob/main/CHANGELOG.md)
 
 ## Usage
 
-Wagtail A/B Testing works with Django 3.2+, Wagtail 4.1+ on Python 3.8+ environments.
+Wagtail A/B Testing works with Django 3.2+, Wagtail 4.1+ on Python 3.9+ environments.
```

Comment:
**Suggestion**: we're testing with Wagtail 5.2 and up, so I think we should update the README too.

```suggestion
Wagtail A/B Testing works with Django 3.2+, Wagtail 5.2+ on Python 3.9+ environments.
```